backend/globaleaks/models/config.py

```diff
@@ -154,9 +154,9 @@ def admin_export(self):
 
 class PrivateFactory(ConfigFactory):
     non_mem_vars = {
+        'acme_accnt_key'
```

Comment:
good to use accnt.

in general let's use as this is only standard abbreviated forms and avoid to use the others.